java/arcs/sdk/android/storage/ServiceStore.kt

```diff
@@ -94,13 +100,18 @@ class ServiceStore<Data : CrdtData, Op : CrdtOperation, ConsumerData>(
     private val connectionFactory: ConnectionFactory,
     private val coroutineContext: CoroutineContext
 ) : ActiveStore<Data, Op, ConsumerData>(options), LifecycleObserver {
+    private val log = TaggedLog { "ServiceStore(${options.storageKey})" }
     private val scope = CoroutineScope(coroutineContext)
     private var storageService: IStorageService? = null
     private var serviceConnection: StorageServiceConnection? = null
-    private val sendQueue = SendQueue()
+    private var channel = BroadcastChannel<suspend () -> Unit>(100)
```

Comment:
The docs for BroadcastChannel are confusing, it says it is non-blocking, but the docs for send() claim that when the buffer is full, it blocks.   I don't think we'll normally come close to hitting the buffer limit, but my Q is, what is the implications if send() blocks, will something else in the stack be frozen?

Reply:
It doesn't block.. it suspends, which is slightly different I guess.  In this scenario, it's only used to send stuff to/from the storage service, so it should be fine